tests/Doctrine/Tests/Models/Upsertable/JoinedInheritanceRoot.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\Models\Upsertable;
+
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\DiscriminatorColumn;
+use Doctrine\ORM\Mapping\DiscriminatorMap;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\GeneratedValue;
+use Doctrine\ORM\Mapping\Id;
+use Doctrine\ORM\Mapping\InheritanceType;
+use Doctrine\ORM\Mapping\Table;
+
+/**
+ * @Entity
+ * @Table(name="joined_inheritance_root")
+ * @InheritanceType("JOINED")
+ * @DiscriminatorColumn(name="discr", type="string")
+ * @DiscriminatorMap({
+ *      "child" = "JoinedInheritanceChild",
+ *      "writable" = "JoinedInheritanceWritableColumn",
+ *      "nonWritable" = "JoinedInheritanceNonWritableColumn",
+ *      "nonInsertable" = "JoinedInheritanceNonInsertableColumn",
+ *      "nonUpdatable" = "JoinedInheritanceNonUpdatableColumn"
+ * })
+ */
+#[Entity]
+#[Table(name: 'joined_inheritance_root')]
+#[InheritanceType('JOINED')]
+#[DiscriminatorColumn(name: 'discr', type: 'string')]
+#[DiscriminatorMap(['child' => JoinedInheritanceChild::class, 'writable' => JoinedInheritanceWritableColumn::class, 'nonWritable' => JoinedInheritanceNonWritableColumn::class, 'nonInsertable' => JoinedInheritanceNonInsertableColumn::class, 'nonUpdatable' => JoinedInheritanceNonUpdatableColumn::class])]
```

Comment:
You can use linebreaks here, can't you?

Reply:
Then all PHP file will be not valid for < PHP 8.0 (multiline attribute), it's ok?)

Reply:
Oh right, please ignore my comment then.